setup.rst

```diff
@@ -22,7 +22,10 @@ Before creating your first Symfony application you must:
 
 Optionally, you can also `install Symfony CLI`_. This creates a binary called
 ``symfony`` that provides all the tools you need to develop and run your
-Symfony application locally.
+Symfony application locally. Also, you may install the `PCNTL`_ extension to
+use some features like signal handling in
+:doc:`the Messenger component </messenger>` and when using
+:ref:`signals in commands <console_signal-event>`.
```

Comment:
I don't think this is needed. There are more "optional extensions" in Symfony components. We should document these when you need it (like you did in this PR), but we shouldn't mention it in the general set-up guide (set-up should be quick and simple)

Reply:
I agree ... the `setup.rst` article is both special and very important. Every item we add to the "list of things to check" creates friction for new people using Symfony.

Given that this PCNTL extension is only needed in some very small and specific parts of some Symfony components, I wouldn't mention it in this article. Thanks.

Reply:
You're right, that's removed. Thank you for the review!